tox.ini

```diff
@@ -42,14 +42,23 @@ description =
     numpy117: with numpy 1.17.*
     numpy118: with numpy 1.18.*
     numpy119: with numpy 1.19.*
+    numpy120: with numpy 1.20.*
+    numpy121: with numpy 1.21.*
+    numpy122: with numpy 1.22.*
+    numpy123: with numpy 1.23.*
     scipy12: with scipy 1.2.*
```

Comment:
Add numpy 1.24, scipy 1.10 and astropy 5.2 here?

Reply:
Yeah I don't think this branch is up to date with main

Reply:
I also think the merge conflict resolution went awry, some the the commits done afterwards make changes that were already on main (codestyle and passenv)